awx/playbooks/project_update.yml

```diff
@@ -1,6 +1,9 @@
 ---
 # The following variables will be set by the runner of this playbook:
-# project_path: PROJECTS_DIR/_local_path_
+# projects_root: Global location for caching project checkouts and roles and collections
+#     should not have trailing slash on end
+# local_path: Path within projects_root to use for this project
+# project_path: projects_root/local_path
```

Comment:
My intent here is that `project_path` will be removed, replaced with `{{projects_root}}/{{local_path}}` in the playbook. After that, the playbook will no longer document this variable, or have it passed in extra_vars.

I just didn't want to do that in this diff, but I could be convinced otherwise.